### PR TITLE
Add markdown comments to llama2_ec2 tutorial.

### PR DESCRIPTION
**Pull request by rohinb2**
chatted with @mkandler, he's gonna do some magic s.t. comments within `if __name__ == "__main__"`: are automatically unindented and used as text blocks, since this is such a common paradigm.

### CODE REVIEW

**Comment by rohinb2** [comment]
* **#547** <a href="https://app.graphite.dev/github/pr/run-house/runhouse/547?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @rohinb2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: examples/llama2/llama2_ec2.py [hunk 1]
```diff
@@ -1,7 +1,28 @@
+# # Llama2 13B Chat Model Inference on AWS EC2
+
+# This example demonstrates how to deploy a Hugging Face model on AWS EC2 using Runhouse. The model is a
+# conversational model called Llama-2-13B.
+#
+# First, you should install the required dependencies.
+#
+# Optionally, set up a virtual environment:
+# $ conda create -n llama-demo-apps python=3.8
```
Comment (mkandler) [inline examples/llama2/llama2_ec2.py:9]: Can you wrap the code here like so?
```suggestion
# ```shell
# $ conda create -n llama-demo-apps python=3.8
# $ conda activate llama-demo-apps
# ```
````

Same with any code within the comments that shouldn't run with the file (line 13 as well)

Comment (rohinb2) [reply examples/llama2/llama2_ec2.py:9]: sounds good

**Comment by mkandler** [comment]
Not a formatting issue, but it would be nice to add h2's (##) before a couple of the comment paragraphs - they'll show up in the table of contents and make the page a little easier to explore

**Review by mkandler — APPROVED** [review]
Looks good to me, might update the file name later but still figuring out how to do this mapping from here to the website.

I'll get on the parsing!

**Review by dongreenberg — APPROVED** [review]

**Comment by rohinb2** [comment]
### Merge activity

* **Mar 4, 6:17 PM EST**: @rohinb2 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/run-house/runhouse/547).

**Pull request merged**